src/main/java/uk/gov/justice/digital/delius/data/api/ManagedOffender.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Data;
+import java.time.LocalDate;
+
+
+@Data
+@Builder
+public class ManagedOffender {
+    @ApiModelProperty(required = true)
+    private String staffCode;
+    @ApiModelProperty(required = true)
+    private Long offenderId;
+    @ApiModelProperty(required = true)
+    private String nomsNumber;
+    @ApiModelProperty(required = true)
+    private String crnNumber;
+    @ApiModelProperty(required = true)
+    private String offenderSurname;
+    @ApiModelProperty(required = true)
+    private boolean isCurrentRo;
```

Comment:
Minor; might be worth expanding this `isCurrentRo` to `isCurrentResponsibleOfficer` they are a bit cryptic to a newbie